release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1379,7 +1384,17 @@ As part of the continued deprecation of the Operator package manifest format, th
 
 *Node*
 
+* Previously, the CRI-O logs did not contain information about the source where images were pulled from. With this fix, the log pull source is added to the info level info-level CRI-O logs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1881694[*BZ#1881694*])
+
+* Previously, when pods were created and deleted rapidly, a pod might not have enough time to complete the pod sandbox creation before the pod starts deletion. As a result, pod deletion could fail with a `ErrCreatePodSandbox`error. This error is now ignored if a pod is terminating. As a result, pod termination no longer fails if a pod could not complete the pod sandbox creation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908378[*BZ#1908378*])
+
+* Previously, the Machine Config Operator (MCO) does not accept *trace* as a valid log level. As a result, the MCO cannot provide a method to enable trace-level logging, even though CRI-O supports it. The MCO was modified to support the *trace* log level. As a result, users can see a trace log level through the MCO configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930620[*BZ#1930620*])
+
+* Previously, the kubelet tried to get the status of images that are not completely pulled. As a result, `crictl` reports a `error locating item named "manifest"` error for these images. CRI-O was changed to not list images that do not have a manifest. As a result, `crictl` no longer reports these errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942608[*BZ#1942608*])
+
+* Previously, outdated status messages were not removed. Because of this, the Machine Config Operator might not be able to locate the proper machine config pool. A cleanup function was added to limit the number of statuses. As a result,  the MCO keeps at most 3 different kubeletConfig status. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1950133[*BZ#1950133*])
```

Comment:
```suggestion
* Previously, outdated status messages were not removed. Because of this, the Machine Config Operator (MCO) was sometimes unable to locate the proper machine config pool. With this release, a cleanup function is added to limit the number of statuses. As a result,  the MCO keeps at most 3 different kubeletConfig status. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1950133[*BZ#1950133*])
```